coordinator/src/main/scala/com/socrata/datacoordinator/service/ServiceConfig.scala

```diff
@@ -22,6 +26,7 @@ class ServiceConfig(val config: Config, root: String, hostPort: Int => Int) {
   val commandReadLimit = config.getBytes(k("command-read-limit")).longValue
   val allowDdlOnPublishedCopies = config.getBoolean(k("allow-ddl-on-published-copies"))
   val instance = config.getString(k("instance"))
+  val collocationGroup = Try(config.as[Set[String]](k("collocation_group"))).getOrElse(Set.empty)
```

Comment:
What is this config used for?  Not a big fan of silently throwing away the config if it can't be parsed.